components/css/global/_base.scss

```diff
@@ -10,77 +10,66 @@ body{
 	body{
 		font-family:$font-arial;
 		font-size:1em;
-		line-height:24/16*1em;
+		line-height:calc-toEm(24);
 	}
 }
 @media (max-width:767px){
 	body{
-		font-size:18/16*1em;
-		line-height:24/18*1em;
+		font-size:calc-toEm(18);
+		line-height:calc-toEm(24);
 	}
 }
 h1{
 	padding-top:0;
 	padding-bottom:0.25em;
-	font-family:$font-archivo-narrow;
-	font-weight:bold;
+	font-family:$font-champion-welter;

```

Comment:
I don't think our template is currently loading in champion. We should probably add this to the _head HTML module before merging.

Reply:
This is kind of up in the air anyway until we get the font situation figured out anyway.